public/js/santaAdmin.js

```diff
@@ -73,8 +73,19 @@ $(async function() {
   $.getJSON(`api/forbidden?groupId=${groupId}`, function(result) {
     // TODO make this beautiful
     result.forEach((pair, index) => {
-      $('#forbiddenPairsTable tbody').append(`<tr><td><b>${++index}</b></td><td>${pair.forbiddenPair1}</td><td>${pair.forbiddenPair2}</td></tr>`);
+      $('#forbiddenPairsTable tbody').append(`<tr><td><b>${++index}</b></td><td class="pairColumn1">${pair.forbiddenPair1}</td><td>${pair.forbiddenPair2}</td><td><i class="buttonDelete bi bi-trash" style='color:red'></i></td></tr>`);
     });
+    $('.buttonDelete').on('click', function ()  {
+        const pair = {
+              forbiddenUser1: $('#forbiddenUser1').val(),
+              forbiddenUser2: $('#forbiddenUser2').val()
+            };
+            $.post(`api/remove`, { forbiddenPairId: $(this).parents('.pairColumn1').html() }, result => {
+                  showAlert(true, result.success);
```

Comment:
what's this used for?

Reply:
We use to display an alert window that contains a specified message

Reply:
Did we agree on that? That was not part of the task as far as I recall.